scripts/remove-unregistered-users.ts

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import { resolve } from "path";
+
+import admin from "firebase-admin";
+
+import {
+  checkFileExists,
+  initFirebaseAdminApp,
+  makeScriptUsage,
+  parseCredentialFile,
+} from "./lib/helpers";
+
+const usage = makeScriptUsage({
+  description: `Bulk remove of user accounts that exist in the auth but are not in the firestore users collection`,
+  usageParams: `CREDENTIAL_PATH`,
+  exampleParams: `fooAccountKey.json`,
+});
```

Comment:
[nit] No need for the backtick quotes:

```suggestion
const usage = makeScriptUsage({
  description: 'Bulk remove of user accounts that exist in the auth but are not in the firestore users collection',
  usageParams: 'CREDENTIAL_PATH',
  exampleParams: 'fooAccountKey.json',
});
```

Reply:
Fair enough. I'll replace them.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1489/commits/9e681f33610b17a47d185592323c0fb8443f6785